ui/app/pages/swaps/select-quote-popover/sort-list/sort-list.js

```diff
@@ -171,6 +171,6 @@ SortList.propTypes = {
   quoteDataRows: PropTypes.arrayOf(QUOTE_DATA_ROWS_PROPTYPES_SHAPE).isRequired,
   sortDirection: PropTypes.number.isRequired,
   setSortDirection: PropTypes.func.isRequired,
-  sortColumn: PropTypes.string.isRequired,
+  sortColumn: PropTypes.oneOfType([null, PropTypes.string]).isRequired,
```

Comment:
... does this work? :thinking: `null` isn't a type...

Reply:
One alternative would be making it not required, and setting a default value of `null` instead. That would get the same intended effect I think.

Reply:
done in cf1480a9b